src/domains/Feedback/ReportForm/ReportForm.stories.tsx

```diff
@@ -16,18 +16,17 @@ export default {
 const dummyBugReportForm = {
     title: 'Bug Report Form',
     mainDescription:
-        'Let us know what went wrong during your virtual town hall experience. We strongly appreciate your time to complete this form',
-    icon: <BugReportIcon />,
+        'Let us know what went wrong during your virtual town hall experience. We strongly appreciate your time to complete this form.',
 };
 
 export function Basic() {
     return (
         <Container maxWidth='sm'>
             <Component
                 title={dummyBugReportForm.title}
-                mainDescription={dummyBugReportForm.mainDescription}
-                icon={dummyBugReportForm.icon}
+                description={dummyBugReportForm.mainDescription}
                 reportType='Bug'
+                townhallId={faker.random.alphaNumeric(12)}
             />
         </Container>
     );
```

Comment:
Change the story to take advantage of storybook more as shown here https://github.com/c2c-project/prytaneum-client/blob/dev/src/components/Chat/Chat.stories.tsx

```ts
interface Props { title: string, mainDescription: string, reporttype: 'Bug' | 'Feedback', townhallId: string }
const Template: Story<Props> = (props) => <Component {...props  />;
export const Bug = Template.bind({});
Bug.args = {
    title: 'title',
    mainDescription: 'asdf',
    reportType: 'bug',
    townhallId: '123'
}
```

Also remove `Container` component since you are centering via the storybook layout unless it's necessary as part of the minimum working example.

Reply:
Without the container, the report form story looks like this. Is this fine?

![reportForm](https://user-images.githubusercontent.com/43712830/107079115-3c5da680-67a4-11eb-9a3d-30365cfd152c.png)

Reply:
Did you add `parameters: { layout: 'centered' }` to the default export?

```ts
export default {  /** other storybook stuff **/ parameters: { layout: 'centered } }
```

Reply:
yes, I did